lib/elixir/test/elixir/test_helper.exs

```diff
@@ -1,4 +1,10 @@
-ExUnit.start [trace: "--trace" in System.argv]
+exclude =
+  case :erlang.system_info(:otp_release) do
```

Comment:
I think we can do this:

```elixir
otp_release = List.to_integer :erlang.system_info(:otp_release)
ExUnit.start [exclude: [:otp_release], include: [otp_release: otp_release], trace: "--trace" in System.argv]
```

And for the tests that do not work on 19, we can do: `@tag otp_release: 18`.

What do you think?